### PR TITLE
FRONT-2432: Blockquote Pattern.

### LINKED ISSUE #8: Refactor blockquote pattern with BCL component

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: templates/patterns/blockquote/pattern-blockquote.html.twig [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+ * @file
+ * Blockquote.
+ */
+#}
+{% include '@oe-bcl/bcl-blockquote/blockquote.html.twig' with {
+  'quote': quote,
+  'attribution': attribution,
+  'cite': cite,
+  'alignment': alignment
+} %}
```
Comment (drishu) [inline templates/patterns/blockquote/pattern-blockquote.html.twig]: add a newline

**Review by drishu — CHANGES_REQUESTED** [review]
we also need to update templates/paragraphs/paragraph--oe-quote.html.twig to send parameters quote, attribution, cite and alignement

**Review by Maxfire — DISMISSED** [review]
Looks good and working nicely. Greetings.

**Review by drishu — APPROVED** [review]

**Comment by cmarcmar** [comment]
Looks good, working as expected.

**Review by cmarcmar — APPROVED** [review]

**Pull request merged**